src/common.tests/TestData/simpleconfig.yml

```diff
@@ -0,0 +1,26 @@
k5SFJ1bmlQelRRUApRS3VWVHJrN3VBTDBnZEtCZWUwWXJ6eG1ZV3dmN0JNSFJrdzBYYlVEYmRJNlA3NmZPV1QvUTY2SFhwMERBODNhClJEUy9ldXRDYXFvaUsrWTZPV3lia0RqSEx4UzZJMVpxRTVNWHVZZyt4alRLM0hFQUZheGVxUEsample9aa0hRd2cKeEtlckpBeEpWWGUwR3ZhdVRoTmlOYzk2MHljaUFnTDgvR2krNEtNdGI1YWdsb2tWaWhwYUlIQzVVKzRZcmhUTQp0RHlqOEtwV2I0Uno3UEpjVUlaYk44NlJ5eEc0anR3blNDUGdJOVZEMjR1SVI4MHovWThtSng2dXNDbGZQTERzCml6NlVKeCt2UFBwcythWVlmQmRrc3RKTGJuMUlrMjY4WTRKVit0R1ZkaTl1UkcySFExWkx4ZksxQmNaS3dtU3IKOFRRQklsbE5FY0g1Y2QwcVJkRmFtMk56d0hiR1gwVVI0RndBcVFmTHRFYmt6RFNMRGZlV05YSFNIVUk2TmFoYwpGbms1TDFtcnJTeFRqb0VvT0ZmQUFHZXk4ZUNuTitCRU10SUpHaGhyOUhTaitENitDVUVDZ2dFQkFOY2F1d2pxCk1lUVU0Q3NNN3B2THZLNWxEcitjZHA1NmRwN2d1RjRKcnVGWDdhSFdod3htMlFKM3FrbHUwbVl1Z0FVRjg3NXgKaE5kNVRqZWY3Z0FwaE9FVWtzM0diNlVzTkdHVkhoUXlaUVNQRm1pQzBLbHBFbWtFejVVcy84MjdoY3VKemNadgo5SjRDQlBCTnQ0eFVHL2diRGhKTDRsWm9Ta3JueGoxOFhZRU9hS1pMSDIxZWlwRCtNMWhMOG11QmpiR0Evb2dZCldqUjBycTB2eTNJcndZR2FEWUdDajFic2xLNTlyMjk1WlpkVG9sRjgzcThIQUdpUmZXYXNXZ05GUDRDaXJoMjUKOUk2cmxKNlNrVFIvK0krK0ZjNU45R3htc3EzcnNyTUVVakQxSzZkY1g1dUVwWlhFWURFamdRbk5JRUd1b05teQo2YU82LzZCVGJ1NEh6ckVDZ2dFQkFPZHJqNWZVQjhJaFlVSms0ejlyOWZHV24zeGpRd3Y1ZnZ5Vm5oUlk2T05MCnYvVk1uK3RCMGMrSjFDbHNnTWdjYUErMjBrTWNRN29HVHcwU3duQWxtZlNNVGZ0MjZpazlPMG15ODZKb0orQlAKaElYeCtRcHJjUGVZeUV3Mis4Z1ZNNTZHc1I4Y2xZeitJZXF4cS9ydXREWEI0Mk5hTnp4QTdjUWUvbWhDd2p0Ugp4TVBWNElVWEZEM2lEL1ByM1hsWVRhOTAvRmtBMkEySE44cUl1VnBScmMvZ2VBc2tNT2FGU1dodnVVMUpaYlFOCmZYcUpjeTVXVkJvL2d2Z2pDcDcrZDViS3A3L0hDREpndWRxTEFOa0lIMnVJd09xSSswcUlkemZicnlwakNvQi8KamtubCttcmRGdGdnNkUrNW40RGdOODFvMzhuV3NjSWVHbTAxVnB0UENjY0NnZ0VBSkQrWTdUT3FIS2wzN1VsTQpuR3lFNjA2VVlhM1lQM1Nic01KbHoxTUthV2NMNEkzcVRDaTR3VFd4T0phNFZnZllBd0N4eHBJU3RNejRaVjlLCkdEVG04NWNuOWcxT215TUw0TFhrVDdyK1V2cFo2RnJERndsaDFOaStzb0RXeXJDakoxMmRrNmNpTGM1aFRxMksKRk5mSHJ0UEFWMHo2WTExSHg3K3ZKSVZrckc3bEs4RFZGdlVybkt0NXM1TjQwcTBPVjY5Sk5WNEZOWG5FUmVNYwpBYTFsK3BVZldZZks1L2NPUnlsZSt3WVdISjQzYUV3R3FyQWZYWC9IQmhwZFFOZldqN0xhYkovOW1sMUh1a0V3CnZ3d2hzdmR4YlVhdHZlYkFqWlora3JSVHRYUmJXdVN5Sjk3K21ISnpVMVNNVmo1L0JSS01nS01weERHckNqV0YKS0ErZ3dRS0NBUUJTVlA0ZE1oNGdmdWV2UzExZDd2YnRJSDFWTDZ0bTB6ak9mQnhsZU14Q01XejF4SEVRcmJZTApkQ203SXhNZUhwMFhHaC9zS0Y0Z0RLYUxMQ1ZvNkxmQlBUbHhRWGxGZXJkaEhQMTR4ajJKbkVDMXRzUzNaRW9vCmpINldQdW1NNFo0ZnhDTzN3ZCtpN2JiN1hJS25IODJWWW9tWGl6MGpmcVlZcEQ0MnNhbVFtU0s3MlplclFlNjkKcFhPdmdEYW5PVGRLT1FDZVdhS0VEWnRYdnhUU2dVU3BORjhBOTdzK0NDVjV0bVhpalQ1VnRJYnoxWHJZL1hXbQpjUG9UZGNRTmxOTFBId0pYUkdReUtCOTNGUVVVTWNPYjZtZFlDYXY5ZEZlVjA5Nlo0ZXFXbE1yMmw0OWovWXhuCkVDNFdUeC9rSmVFMGZ4OC9HOWJhRFFPeDNzd1ZMU1Y1QW9JQkFRRE1KemFrcmw4d1hDamlhbWV4YUtwRVRFL2cKaENmN2h5ZnFLcEJ3SDJ5VXFHbUZWaVdiOTFtSXp4N25FbjdaQnhaNnVlSGhSTWNWNEk3WmlhT3lwTDVkQWtSMQpGRndzY3VQNGNweXExZVFVQU9LWU1oY0JNeEsyTVBLZ1d5NCtjU0JuNStITitYdEF5UysxUlM2cHd4THo5QWVuCktma3NNQ1FYU0ZFZzViRWtBRlRsTENMNEw5cXdHNmpOV09zZlhBR29YaXJCRjYwOFpHWjhqd0l2dHFiT3AxankKUEZ6bmhvUFFvZnNGSGN3RVdrWXdvbHNpL21yb0RWUGNUSlBYVzg2NTQ2OHh1UzdJLzAxNzNZZGVYRVJQWVpaMAo0amlTR25mSXg1NGZVYW5UVUhpNENpbytKbHRXdTZ6N1B1L3RCNjF5WjdaZ3lsdGVMbFlpVDBHTVFHTG0KLS0tLS1FTkQgUlNBIFBSSVZBVEUgS0VZLS0tLS0K
```

Comment:
💡 Same here correlating to thought with this: https://github.com/Azure/Bridge-To-Kubernetes/pull/258/files#r1180809770